src/redux-middleware/__tests__/pullQueue.ts

```diff
@@ -6,6 +6,8 @@ import * as dexie from '../../data-providers/dexie'
 import getContext from '../../data-providers/data-helpers/getContext'
 import { DataProvider } from '../../data-providers/DataProvider'
 
+// mock getUserRef (firebase's database.ref)
+jest.mock('../../util/getUserRef')
```

Comment:
Should this be mocked as logged in or logged out? Does it vary per test?

Reply:
No, it doesn't vary per test. Besides, it works even without the mock (with the newly added support for null values) so we can safely remove it now.
Whether it should be mocked as `logged in` or `logged out` is something that I'd let you answer since you're more familiar with this function

Reply:
I think it should be mocked as logged in since it's only mocked in the firebase tests now.

Reply:
Okay, I've updated the test